test/codegen/golden/expected/url_shortener/app/routers/url_mappings.py

```
@@ -0,0 +1,46 @@
+@router.post("/shorten", status_code=201)
+async def shorten(
+    body: UrlMappingCreate,
+    session: AsyncSession = Depends(get_session),
+) -> UrlMappingRead:
+    svc = UrlMappingService(session)
+    return await svc.shorten(body)
+
+@router.get("/{code}", status_code=302)
+async def resolve(
+    code: str,
+    session: AsyncSession = Depends(get_session),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)